### PR TITLE
Add edx-enterprise-data installation instructions

### PR DESCRIPTION
**Pull request by raq929**
Instructions for installing edx-enterprise-data and links to the frontend enterprise app.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -45,21 +48,32 @@ Getting Started
 
       $ make test.run_elasticsearch
 
-3. Create a user and authentication token. Note that the user will be
+#. Create a user and authentication token. Note that the user will be
    created if one does not exist.
 
    ::
 
        $ ./manage.py set_api_key <username> <token>
```
Comment (binodpant) [inline README.rst:56]: I always wondered what this token here should be, maybe can add a line? I have been entering garbage values for it like '1alkdjflsd'

File: README.rst [hunk 2]
```diff
@@ -45,21 +48,32 @@ Getting Started
...
 
+Development with edx-enterprise-data
+------------------------------------
+If you need to make changes to ``edx-enterprise-data`` and have them reflected when you run the ``edx-analytics-data-api`` server,
+you can follow these steps. If you do not intend to make changes to ``edx-enterprise-data``, you can skip this section.
+
+#. Recommended: Install this repo into a subfolder of your working directory. Within that subfolder create an ``src`` folder.
+#. Clone the `edx-enterprise-data <https://github.com/edx/edx-enterprise-data>`_ repo into the ``src`` folder.
+#. ``cd`` into your ``edx-data-analytics-api`` folder and activate your virtualenv.
+#. Run ``/src/edx-enterprise-data``.
```
Comment (binodpant) [inline README.rst:74]: is this `/edx/src/edx-enterprise-data` when in the shell?

File: README.rst (hunk 1 above)
Comment (raq929) [reply README.rst:56]: Ha, I'm not exactly sure what it is myself; I assume it's a token for API access? imho we should be generating that ourselves and giving it to the user, not the other way around...

File: README.rst (hunk 2 above)
Comment (raq929) [reply README.rst:74]: Not for me. My file structure is:
```
edx
  |
  analytics
    |
    edx-analytics-data-api
    src
        |
        edx-enterprise-data
```

Oh, but that should be a pip install command. Let me update.

**Pull request merged**